pgmpy/models/MarkovModel.py

```diff
@@ -676,3 +676,30 @@ def get_partition_function(self):
             raise ValueError('Factor for all the random variables not defined.')
 
         return np.sum(factor.values)
+
+    def copy(self):
+        """
+        Returns a deep copy of this Markov Model.
+ 
+        Returns
+        -------
+        MarkovModel: Deep copy of this Markov model.
```

Comment:
I know what deepcopy is. It is just that I find it a bit misleading. There is no need to change it if you find it useful.